Javatrix.java

```diff
@@ -477,5 +477,31 @@ public Javatrix times(double scalar)
         }
         return actual;
     }
+    /**
+     * Return calling matrix A = A \ B.
+     *
+     * @param actualB calling matrix
+     * @return A \ B
+     */
+    public Javatrix arrayLeftDivideEquals(Javatrix actualB)
+    {
+        m = actualB.getRowDimension();
+        n = actualB.getColumnDimension();
+        double a;
+        double b;
+        double c;
+        for (int i = 0; i < m; i++)
+        {
+            for (int j = 0; j < n; j++)
+            {
+                a = this.get(i, j);
+                b = actualB.get(i, j);
+                c = b / a;
+                this.set(i, j, c);
+            }
+        }
+        return this;
+    }
+
```

Comment:
@caldwellc1 Sorry about that - eyes playing tricks on me (I'm not doing a very good job of practicing energized work right now...)! Disregard!